cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -28,7 +28,7 @@ subsequent programs in the sequence are executed.
 The control flow of a workflow can be described as a directed, acyclic graph (DAG) of actions.
 To be more precise, we require that it be a series-parallel graph. This is a graph with a
 single start node and a single finish node. As described below, execution can be either
-sequential (the default) or :ref:`concurrent <workflow-concurrent>`, and the graph can be 
+sequential or :ref:`concurrent <workflow-concurrent>` (the default), and the graph can be 
```

Comment:
We can remove this `As described below, execution can be either sequential (the default) or concurrent, and `